src/containers/message/components/ParticipantSelectionTable.js

```diff
@@ -0,0 +1,60 @@
+// @flow
+import { Map } from 'immutable';
+import { Table } from 'lattice-ui-kit';
+
+import ParticipantRow from './ParticipantRow';
+
+import EnrollmentStatuses from '../../../utils/constants/EnrollmentStatus';
+import { PROPERTY_TYPE_FQNS } from '../../../core/edm/constants/FullyQualifiedNames';
+import { COLUMN_FIELDS } from '../../study/constants/tableColumns';
+
+const { PERSON_ID } = PROPERTY_TYPE_FQNS;
+const { ENROLLMENT_STATUS } = COLUMN_FIELDS;
+
+const HEADERS = [
+  {
+    key: 'id',
+    label: 'Participant Id'
+  },
+  {
+    key: 'actions',
+    label: '',
+    sortable: false,
+    cellStyle: { width: '50px' }
+  }
+];
+
+const ParticipantSelectionTable = ({
+  handleOnSelect,
+  participants,
+  targetParticipants
+} :{
+  handleOnSelect :(participants :Map) => void;
+  participants :Map;
+  targetParticipants :Map;
+}) => {
+
+  const participantOptions = participants
+    .valueSeq().filter((participant) => (participant.getIn([ENROLLMENT_STATUS, 0]) !== EnrollmentStatuses.ENROLLED))
+    .map((participant) => ({ id: participant.getIn([PERSON_ID, 0], '') })).toJS();
+
```

Comment:
this filters out enrolled participants. I think it makes sense to send links to participants who are marked as active/enrolled instead

Reply:
Is the filter even necessary? I think we should just let the user pick which participants to send the text to, no filtering.

Reply:
you want to send enrollment links to people that are already enrolled?

Reply:
'Enrolled' status doesn't mean that the participant has enrolled their device. In the current usage it implies that the participant is active (they are able to participate in the data collection process)